lib/events.js

```diff
@@ -46,6 +49,26 @@ EventEmitter.EventEmitter = EventEmitter;
 
 EventEmitter.usingDomains = false;
 
+Object.defineProperty(EventEmitter, 'autoCatch', {
+  get() {
+    return EventEmitter.prototype[kApply] === applyAndCatch || false;
```

Comment:
Why does this need the `|| false`?

Reply:
I think it's a some remnant of old code. I'll check and remove.